static/episodes.json

```
@@ -86,5 +86,110 @@
         "detail": "Domain Sponsor"
       }
     ]
+  },
+  {
+    "id": "2",
+    "title": "Testing Components (SFC)",
+    "time": "1595224800",
+    "length": "33606137",
+    "duration": "56:00",
+    "keywords": "expect exception, testing, frontend, software, web development, programming, code, coding, software engineering",
+    "subtitle": "What are the best practices when testing single file components?",
+    "description": "In this episode, Mirjam Bäuerlein, Roman Kuba, and Vanessa Böhner talked about how to test Single File Components (SFC). They discussed when they 'shallowMount' and 'mount' components, and when they mock or don't mock API requests, stores, or other dependencies. Roman and Mirjam tell stories about performance issues whereas Vanessa never had any working on a micro-frontend project for the last couple of months.",
```

```suggestion
    "description": "In this episode, Mirjam Bäuerlein, Roman Kuba, and Vanessa Böhner talked about how to test Single File Components (SFC). They discussed when they use 'shallowMount' or 'mount' for components, when they mock or don't mock API requests, stores, or other dependencies. Roman and Mirjam tell stories about performance issues whereas Vanessa never had any working on a micro-frontend project for the last couple of months.",
```

In the last sentence it seems like a word is missing? "never had any working on"

---

thanks!

---

wrote that on a new Mac book and I totally forget that even a MacBook has autorrection enabled by default :D